config/linux_gpu.cmake

```diff
@@ -30,6 +30,10 @@
 #  $ cmake ..
 #  $ cmake --build .
 #
+#  or you can specify the particular GPU architecture by
+#
+#  $ cmake .. -DMXNET_CUDA_ARCH=7.0
```

Comment:
@leezu how about a compromise, a short example at the top?

Reply:
@samskalicky I'm totally for highlighting the `MXNET_CUDA_ARCH` option at the top of this file. My concern is more general, in that a few months ago we started of with asking users to specify options via the `-D` arguments to `cmake ..`, but reviewers pushed for using a config file. Now you documented inside the config file how to use the `-D` option.

Thus let's be consistent and highlight the `MXNET_CUDA_ARCH` option, while recommending users to edit the value of `MXNET_CUDA_ARCH` 20 lines below. I'm afraid there may be confusion if we ask inside the config file that users should use the `-D` option to overwrite the config file. WDYT?

We may also add a README file in the `config/` directory and explain that all optiions can be overwritten by passing a new value via `-D` to `cmake`.

Finally, your initial explanation for the `-DMXNET_CUDA_ARCH` option was easy to understand, so it would be great to use it to improve the existing documentation.